subprojects/docs/src/docs/release/notes.md

```diff
@@ -146,6 +146,11 @@ The following are the newly deprecated items in this Gradle release. If you have
 
 ## Potential breaking changes
 
+### maven-publish and ivy-publish mirror multi-project behavior
+
+When using the `java` plugin, all `compile` and `runtime` dependencies will now be mapped to the `compile` scope, i.e. "leaked" into the consumer's compile classpath. This is in line with how
```

Comment:
I'm going to bring this up just one more time... are we **SURE** we want to do this?  It seems to me the behavior is better now than it will be after this change, with compile -> compile and runtime -> runtime.

As for making it consistent with multi-project behavior... consistency is nice, but not if it's consistent bad behavior.

But I also don't know if there's some secondary side effects of this inconsistency that causes bigger problems.